### ISSUE TITLE
Existing window not raised when using SyncteX in Emacs

### ISSUE THREAD

**Issue by sebastinas** [pwmt/zathura#141]
*On GitLab by @agreselin on Aug 5, 2019, 19:45*

---

When a (La)TeX document is compiled and the Zathura window opened for preview, it's raised to the foreground as expected. If now one doesn't close the window, just lets it go in the background and compiles again, that window isn't raised and must instead be brought back to the foreground with some manual operation (e.g. Alt-TAB).

Evince consistently raises its window when used for previewing TeX documents written with Emacs, so I imagined the issue is related to Zathura.

Related Emacs setting:
```
(with-eval-after-load "latex"
  (add-to-list 'TeX-view-program-selection
               '(output-pdf "Zathura")))
```
Sw version:
Emacs 26.2, zathura 0.4.3, girara 0.3.2, Gnome Shell 3.32.2 on Fedora 30 running Wayland.

**Comment by sebastinas** [comment]
Should be fixed in e90755f.

**Comment by sebastinas** [comment]
closed

**Comment by sebastinas** [comment]
closed via commit e90755f06c9cd505296022c9f4abf8bd8c516af5

**Comment by sebastinas** [comment]
mentioned in commit e90755f06c9cd505296022c9f4abf8bd8c516af5
